### PR TITLE
[CI] Add Transifex flow to automatically update translations and templates

### PR DESCRIPTION
**Pull request by AenBleidd**
Signed-off-by: Vitalii Koshura <lestat.de.lionkur@gmail.com>

### CODE REVIEW

File: .github/workflows/transifex.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Install dependencies
+        run: |
+          sudo apt-get -qq update
+          sudo apt-get install -y gettext translate-toolkit transifex-client
+      - name: Configure Transifex
+        run: |
+          echo "[https://www.transifex.com]\n" > ~/.transifexrc
```
Comment (talregev) [inline .github/workflows/transifex.yml:28]: You should remove \n from the end. It already separate the line for each echo.

File: .github/workflows/transifex.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          echo "hostname = https://www.transifex.com\n" >> ~/.transifexrc
+          echo "username = api\n" >> ~/.transifexrc
+          echo "token = ${{ secrets.TRANSIFEX_API_KEY }}\n" >> ~/.transifexrc
```
Comment (talregev) [inline .github/workflows/transifex.yml:31]: Also I believe the `secrets.TRANSIFEX_API_KEY` is empty.

File: .github/workflows/transifex.yml (hunk 1 above)
Comment (AenBleidd) [reply .github/workflows/transifex.yml]: Yes, you're right. Removed

File: .github/workflows/transifex.yml (hunk 2 above)
Comment (AenBleidd) [reply .github/workflows/transifex.yml]: No, it's not. I added it, you will just never see it on CI

**Comment by talregev** [comment]
```bash
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"
```
You should look solution like this: 
https://github.com/marketplace/actions/add-commit

**Comment by talregev** [comment]
I can help you if you want.

**Comment by AenBleidd** [comment]
Thanks, I'm fine

**Pull request closed without merging**